### PR TITLE
Internal 576 fix graphql test client

### PR DESCRIPTION
**Pull request by jgrillo-grapl**
### Which issue does this PR correspond to?
https://github.com/grapl-security/issue-tracker/issues/576

### What changes does this PR make to Grapl? Why?
some updates to make the e2e-test-runner work, fix some logging issues, make `graplctl aws test` exit 0 upon success

### How were these changes tested?
`graplctl aws test`, `make test-e2e`

### CODE REVIEW

File: src/python/grapl-tests-common/grapl_tests_common/setup_tests.py [hunk 1]
```diff
@@ -13,6 +13,7 @@
 DUMP_ARTIFACTS = bool(environ.get("DUMP_ARTIFACTS", False))
 
 LOGGER = get_module_grapl_logger()
+GRAPL_LOG_LEVEL = os.getenv("GRAPL_LOG_LEVEL", "INFO")
```
Comment (wimax-grapl) [inline src/python/grapl-tests-common/grapl_tests_common/setup_tests.py:16]: ?

File: src/python/grapl-common/grapl_common/grapl_logger.py [hunk 2]
```diff
@@ -4,8 +4,8 @@
 import sys
 
 
-def get_module_grapl_logger(
-    default_log_level: str = "INFO", log_to_stdout: bool = False
+def get_module_grapl_logger(  # TODO: delete this function
```
Comment (wimax-grapl) [inline src/python/grapl-common/grapl_common/grapl_logger.py:7]: can we get a proposed solution and/or a stub ticket for this?

i am not attached to this particular solution; I _am_ attached to a mostly-universal way to grab a logger that knows about GRAPL_LOG_LEVEL.

File: src/python/e2e-test-runner/e2e_test_runner/main.py [hunk 3]
```diff
@@ -1,15 +1,32 @@
...
+    LOGGER.info("executing pytest")
+
+    from grapl_tests_common import setup_tests  # import here to limit monkeypatch
+
+    result = setup_tests.exec_pytest()
+
+    LOGGER.info(f"tests completed with status code {result}")
+
+    if result == 0:
+        return  # Lambda runtime will flip out if sys.exit(0) -_-
```
Comment (wimax-grapl) [inline src/python/e2e-test-runner/e2e_test_runner/main.py:30]: LMFAO WHAT

File: src/python/grapl-common/grapl_common/grapl_logger.py (hunk 2 above)
Comment (jgrillo-grapl) [reply src/python/grapl-common/grapl_common/grapl_logger.py:7]: The proposed solution is to adhere as closely as possible to the python documentation. In particular:

> Note that Loggers should NEVER be instantiated directly, but always through the module-level function logging.getLogger(name).

https://docs.python.org/3/library/logging.html#logger-objects

So, everywhere we need a logger, we should do the following:
```python
GRAPL_LOG_LEVEL = os.getenv("GRAPL_LOG_LEVEL", "INFO")
LOGGER = logging.getLogger(__name__)
LOGGER.setLevel(GRAPL_LOG_LEVEL)
LOGGER.addHandler(logging.StreamHandler(stream=sys.stdout))
```
there's nothing wrong with a little repetition here. Making this simple and explicit is much more important than making it convenient and clever.

File: src/python/grapl-tests-common/grapl_tests_common/setup_tests.py (hunk 1 above)
Comment (jgrillo-grapl) [reply src/python/grapl-tests-common/grapl_tests_common/setup_tests.py:16]: Is there a question? I don't understand.

Comment (wimax-grapl) [reply src/python/grapl-tests-common/grapl_tests_common/setup_tests.py:16]: oh duh you do actually use this, okay

**Review by wimax-grapl — APPROVED** [review]

**Pull request merged**